recipes/liquid-dsp/all/conanfile.py

```diff
@@ -0,0 +1,170 @@
+        if self.settings.os == "Macos" and not self.options.shared:
+            return "libliquid.a"
+        if self.settings.os != "Windows":
+            return self._target_name
+        return "libliquid.lib"
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def build_requirements(self):
+        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
+        if self.settings.compiler == "Visual Studio":
+            self.build_requires("mingw-w64/8.1")
+            self.build_requires("automake/1.16.3")
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            destination=self._source_subfolder,
+            strip_root=True,
+        )
+
+    def _patch_sources(self):
+        if self.settings.os == "Windows":
+            for patch in self.conan_data["patches"][self.version]:
+                tools.patch(**patch)
+
+    def _gen_link_library(self):
+        if self.settings.compiler != "Visual Studio" or (not self.options.shared):
+            return
+        self.run("cmd /c generate_link_library.bat")
+        with tools.chdir(self._source_subfolder):
+            self.run(
+                "{} /def:libliquid.def /out:libliquid.lib /machine:{}".format(
+                    os.getenv("AR"), "X86" if self.settings.arch == "x86" else "X64"
+                ),
+                win_bash=tools.os_info.is_windows,
+            )
+
+    def _rename_libraries(self):
+        with tools.chdir(self._source_subfolder):
+            if self.settings.os == "Windows" and self.options.shared:
+                os.rename("libliquid.so", "libliquid.dll")
+            elif self.settings.os == "Windows" and not self.options.shared:
+                os.rename("libliquid.a", "libliquid.lib")
+            elif self.settings.os == "Macos" and not self.options.shared:
+                os.rename("libliquid.ar", "libliquid.a")
+
+    @contextmanager
+    def _build_context(self):
+        if self.settings.compiler == "Visual Studio":
+            env = {
+                "CC": "gcc",
+                "CXX": "g++",
+                "LD": "ld",
+                "AR": "ar",
+            }
+            with tools.environment_append(env):
+                yield
+        else:
+            yield
+
+    @contextmanager
+    def _msvc_context(self):
+        if self.settings.compiler == "Visual Studio":
+            with tools.vcvars(self.settings):
+                env = {
+                    "CC": "cl -nologo",
+                    "CXX": "cl -nologo",
+                    "AR": "lib",
+                    "LD": "link",
+                }
+                with tools.environment_append(env):
+                    yield
+        else:
+            yield
+
+    def build(self):
+        self._patch_sources()
+        ncpus = os.cpu_count()
+        ncpus = ncpus if ncpus else 1
```

Comment:
```suggestion
        ncpus = tools.cpu_count()
```

https://docs.conan.io/en/latest/reference/tools.html#tools-cpu-count